main.go

```diff
@@ -167,15 +167,15 @@ func main() {
 		DefaultWorkloadClusterNamespaces: defaultNamespaces,
 	}
 
-	eventsLoggerSecret := eventsloggersecret.Reconciler{
```

Comment:
You can remove this change I guess :D

Reply:
WDYM ? We still need a reconciler for the secrets and it's initialized here

Reply:
Thé move of this réconcilier specifically, but it's not important